crates/cdk-integration-tests/tests/integration_tests_pure.rs

```diff
@@ -565,7 +565,7 @@ async fn test_swap_overpay_underpay_fee() {
         .expect("Failed to create test mint");
 
     mint_bob
-        .rotate_keyset(CurrencyUnit::Sat, 32, 1)
+        .rotate_keyset(CurrencyUnit::Sat, (0..32).map(|n| 2u64.pow(n)).collect(), 1)
```

Comment:
We do this in multiple places both in tests and app code. We should make it a fn so we can comment what it is an reuse it.